README-CONAN.md

```diff
@@ -117,25 +202,208 @@ os_build=Windows
+| | compiler.version=14 | compiler.version=15 |
+| | compiler=Visual Studio | compiler=Visual Studio |
+| | os=Windows | os=Windows |
+| | os\_build=Windows | os\_build=Windows |
+
+##### Debug Builds
+
+|| Visual Studio 2015 | Visual Studio 2017|
+|:-------|-------|------|
+| _**conan install .. --profile**_ | msvc2015Debug64 | msvc2017Debug64 |
+| _**cmake**_ | -DCMAKE\_BUILD\_TYPE=Debug | -DCMAKE\_BUILD\_TYPE=Debug |
+| _**profile**_ | build_type=Debug | build_type=Debug |
+| | compiler.runtime=MDd | compiler.runtime=MDd |
+
+##### 32bit Builds
+
+|| Visual Studio 2015 | Visual Studio 2017|
+|:-----------|--------------------|--------------------|
+| _**conan install .. --profile**_ | msvc2015Release32 | msvc2017Release32 |
+| _**cmake -G**_ | "Visual Studio 14 2015" | "Visual Studio 15 2017" |
+| _**profile**_ | arch=x86 | arch=x86 |
+| | arch_build=x86 | arch_build=x86 |
+
+### Modifying build settings in the Visual Studio IDE
+
+**CAUTION** You can change the runtime library in Visual Studio once you open the
+solution generated by CMake.  If you have generated a project to use the dynamic run-time library
+with the profile setting `compiler.runtime=MD` (or MDd), Visual Studio allows you to change that setting.
+However the libraries (provided by conan) to which you link are tied
+to a specific configuration and you will obtain linking errors.
+
+If you want to work with the static runtime library `MT` (or `MTd`), set up the profiles and build in a different directory such as build-static.
+
+### Changing profile settings with the conan command
+
+It is recommended that you create 8 profiles for msvc{2017|2015}{Release|Debug}{64|32}.
+
+You can modify profile settings on the command line.
+The following example demonstrates making substantial changes to profile settings by performing a 32 bit build using Visual Studio 2015 with a 2017 profile!
+
+```bash
+$ conan install .. --profile msvc2017Release64 -s arch_build=x86 -s arch=x86 -s compiler.version=14
+$ cmake         .. -G "Visual Studio 2015"
+$ cmake --build .  --config Release
+```
+
+### CMake/Cygwin Collisions on the build machine
+
+If you have Cygwin installed on your build machine, you may encounter the situation
+that CMake erroneously finds library files in Cygwin directories and adds `c:\\cygwin64\\usr\\include` to the
+compiler header search path.  FindIntl is a prime suspect and believe it's caused by %PATH%.
+
+If necessary, temporarily rename c:\\cygwin64\\usr\\include as c:\\cygwin64\\usr\\uncle to hide those files when working with CMake.
+
+<name id="2-3"></a>
+### 2.3) Cygwin Notes
+
+Do not use conan on the Cygwin Platform.  To build Exiv2 for Cygwin use CMake.  See [README-CMAKE](README-CMAKE)
+
+<name id="2-4"></a>
+### 2.4) MinGW Notes
+
+Team Exiv2 does not provide support for MinGW.
+
+Users have reported success with CMake (without conan) on both msys/1.0 and msys/2.0.
+
+[TOC](#TOC)
+
+<name id="3">
+## 3 Conan Architecture
```

Comment:
Something breaks GitHub's markdown renderer at this point. All the following headings are not rendered properly: https://github.com/Exiv2/exiv2/blob/README-CONAN.md/README-CONAN.md

The markdown looks however perfectly fine to me. Anyone got a clue?

Reply:
I suspect the nested apostrophes have broken their markdown processor.  

```
self.requires('self.requires('zlib/1.2.11@conan/stable')')
```